engine/src/exec-interface-object.cpp

```diff
@@ -2895,9 +2895,9 @@ void MCObject::SetTextFont(MCExecContext& ctxt, MCStringRef font)
 		//   to ensure substacks update properly.
 		// MW-2013-03-21: [[ Bug ]] Unless its the templateStack (parent == nil) in which
 		//   case we don't want to do any font recomputation.
-		if ((gettype() == CT_STACK && parent != nil) || opened)
+		if (gettype() == CT_STACK || opened)
 		{
-			if (recomputefonts(parent -> getfontref()))
+			if (parent != nil && recomputefonts(parent -> getfontref()))
```

Comment:
@livecodefraser is going to turn `MCObject::parent` into an `MCObjectHandle` in #4804, so  this change will cause a merge conflict.